fang/README.md

```diff
@@ -261,10 +290,10 @@ let mut queue = AsyncQueue::builder()
     .build();
 
 // Always connect first in order to perform any operation
-queue.connect(NoTls).await.unwrap();
+queue.connect().await.unwrap();
 ```
 
-As an easy example, we are using NoTls type. If for some reason you would like to encrypt Postgres requests, you can use [openssl](https://docs.rs/postgres-openssl/latest/postgres_openssl/) or [native-tls](https://docs.rs/postgres-native-tls/latest/postgres_native_tls/).
+Encryption is always used with crate `rustls`. We plan to add the possibility of disabling it in the future.
```

Comment:
maybe it can be removed? I think sqlx will work with or without it, no?

Reply:
See [Cargo.toml](https://github.com/launchbadge/sqlx/blob/v0.6.3/Cargo.toml), it seems not to export a no-tls feature. Thats why this comment.